### PR TITLE
Add `stylelint-config-suitcss`.

### PR DESCRIPTION
**Pull request by lucasmazza**
This adds https://github.com/suitcss/stylelint-config-suitcss as an available plugin

### CODE REVIEW

**Comment by leonelgalan** [comment]
You should add the plugin to the README as well, so when the PR gets merged it's added to https://github.com/gilbarbara/codeclimate-stylelint/blob/master/README.md#plugins with the rest.

**Comment by Supernats** [comment]
We would also love to see this added. Is there anything I can do to help move this along? I'd be happy to add to the README.

File: README.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ To run the tests, cd into `tests/` and run `sh setup.sh`
 ### Config
 - [stylelint-config-standard](https://github.com/stylelint/stylelint-config-standard/): The standard shareable config for stylelint
 - [stylelint-config-wordpress](https://github.com/ntwb/stylelint-config-wordpress/): WordPress CSS Coding Standards shareable config for stylelint
+- [stylelint-config-wordpress](https://github.com/suitcss/stylelint-config-suitcss): SUIT CSS config for stylelint
```
Comment (leonelgalan) [inline README.md]: Wrong plugin name (in the link's text)

Comment (lucasmazza) [inline README.md]: 🙈 thanks for spotting it

**Comment by gettinToasty** [comment]
seconding @Supernats' request here, are there any more blockers for this?

**Comment by Supernats** [comment]
@gilbarbara do you have issues with this PR in its current state? If so, could you highlight them?

@dblandin do you have merge rights for this repo? Would you be comfortable merging it in?

**Comment by dblandin** [comment]
@Supernats Hey! Yes, I do have push access. I can take a look at the open PRs, including this one.

I might split this PR into two commits before merging:

- Fix formatting in README
- Add stylelint-config-suitcss plugin

**Comment by gilbarbara** [comment]
Hey!
Sorry y'all. Github never emailed me about this. :/

**Comment by Supernats** [comment]
Thanks so much, @gilbarbara! This is great.

**Pull request merged**